WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert student progress view

## Changes

Converted student progress to Angular.

## Test

1. Open a run in the Classroom Monitor
2. Go to the "Grade By Team" view
3. Make sure everything works like before
- Find a student
- Sort by column
- Mouseover tooltips
- Clicking on a workgroup
- Make sure things update in real time like student location
- Changing period shows correct workgroups

@breity The sticky header row doesn't stick anymore. Can you see if you can fix this? If not, it's ok if we don't have the sticky functionality anymore since our other views don't have it and we can always add it back when we redesign.

Closes #947
